src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -475,26 +475,91 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
 
   override def isEnabled(subject: WorkbenchSubject): IO[Boolean] = ???
 
-  override def getUserFromPetServiceAccount(petSA: ServiceAccountSubjectId): IO[Option[WorkbenchUser]] = ???
+  override def getUserFromPetServiceAccount(petSA: ServiceAccountSubjectId): IO[Option[WorkbenchUser]] = {
+    import SamTypeBinders._
+
+    runInTransaction { implicit session =>
+      val petServiceAccountTable = PetServiceAccountTable.syntax
+
+      val userTable = UserTable.syntax
+      val userColumn = UserTable.column
+
+      val loadUserQuery = samsql"""select ${userTable.id}, ${userTable.email}, ${userTable.googleSubjectId} from ${UserTable.table} left join ${PetServiceAccountTable.table} on ${petServiceAccountTable.userId} = ${userTable.id} where ${petServiceAccountTable.googleSubjectId} = ${petSA}""" //TODO harmonize the subjectId types
```

Comment:
should not be using `PetServiceAccountTable.table` or `UserTable.column`, I think those are useful in insert, update and delete but not select. 

Instead
```
select ${userTable.id}, ${userTable.email}, ${userTable.googleSubjectId} 
from ${UserTable as userTable} left join ${PetServiceAccountTable as petServiceAccountTable} 
on ${petServiceAccountTable.userId} = ${userTable.id} 
where ${petServiceAccountTable.googleSubjectId} = ${GoogleSubjectId(petSA.value)}
```
and why is this a left join?